com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicSettings.cs

```diff
@@ -174,5 +174,15 @@ public bool ImportCurves
             get { return importCurves; }
             set { importCurves = value; }
         }
+
+        [SerializeField] bool createCurveRenderers;
+        /// <summary>
+        /// If importing of Alembic curves is enabled, also create automatically the AlembicCurveRendering component.
```

Comment:
Suggestion:
`If you enabled the importing of Alembic curves, this method automatically creates the AlembicCurveRendering component.`